packages/realm-web/src/App.ts

```diff
@@ -247,6 +247,17 @@ export class App<
     this.storage.removeUserId(user.id);
   }
 
+  /**
+   * @inheritdoc
+   */
+  public async deleteUser(user: User<FunctionsFactoryType, CustomDataType>): Promise<void> {
+    await this.fetcher.fetchJSON({
```

Comment:
Do we need some sort of error handling here?

Reply:
@kraenhansen What do you think?

Reply:
I mean, if `fetchJSON` throws, then it would probably be clear in the stack trace that there was an issue with `deleteUser`.  Just want to make sure this doesn't fail silently on a network issue

Reply:
Nothing fails silently here. We await the result of `fetchJSON` so any rejection will be propagated as expected.

Reply:
I will leave it as it is.